Examples/Makefile.in

```diff
@@ -1754,8 +1765,8 @@ go: $(SRCDIR_SRCS)
 	if test -f $(SRCDIR)$(RUNME).go; then \
 	  $(GO) $(GOCOMPILEARG) $(SRCDIR)$(RUNME).go; \
 	  if $(GOGCC) ; then \
-	    $(COMPILETOOL) $(GO) -o $(RUNME) $(RUNME).@OBJEXT@ $(GOGCCOBJS) $(OBJS) $(IOBJS); \
-	  elif $(GO12) || $(GO13); then \
+	    $(COMPILETOOL) $(GO) -o $(RUNME) $(RUNME).%@OBJEXT@ $(GOGCCOBJS) $(OBJS) $(IOBJS); \
```

Comment:
No. I made the classical error of updating the generated Makefile, and then
realized that it's generated from Makefile.in, so I copied my changes over
and then revert the %OBJEXT% to o change made by config.status.

Do you want me to rewrite that commit with the correct file or just add a
new commit to fix the bug into the pull request?